src/s3/util/fileViewerManager.ts

```diff
@@ -360,23 +494,27 @@ export class S3FileViewerManager {
     }
 
     public async createTemp(): Promise<string> {
-        this._tempLocation = await makeTemporaryToolkitFolder()
+        this.tempLocation = await makeTemporaryToolkitFolder()
```

Comment:
Are you storing a `tempLocation` and a `_tempLocation` value? Unsure if this switch was intended.

Reply:
that was just to differentiate it from the getter